electrotrans-web-ui/src/main/java/decavun2/config/personnel/PersonWebUiConfig.java

```diff
@@ -64,7 +67,8 @@ private PersonWebUiConfig(final Injector injector, final IWebUiBuilder builder)
     private EntityCentre<Person> createCentre(final IWebUiBuilder builder) {
         final String layout = cell(
                 cell(cell().repeat(2).layoutForEach(CELL_LAYOUT).withGapBetweenCells(MARGIN))  // row 1 -> 1, 2
-               .cell(cell().repeat(2).layoutForEach(CELL_LAYOUT).withGapBetweenCells(MARGIN)), // row 2 -> 3, 4
+                .cell(cell().repeat(2).layoutForEach(CELL_LAYOUT).withGapBetweenCells(MARGIN))  // row 2 -> 3, 4
```

Comment:
@Sukhorskyy would it be possible to simplify layout creation with `LayoutComposer`? By using  `mkGridForCentre`, for example.

Reply:
Of course, we can simplify this part of the code, by using a `LayoutComposer` as you suggested, which is a good idea,  but I don't see an urgent need for it. The approach that Yurii used in his code is also valid and works correctly. In a further development I support Stefan and also recommend  you to use this approach with the `LayoutComposer`, because it`s simplier.